### PR TITLE
Added Burnt Sienna theme for xpui

### PR DESCRIPTION
**Pull request by pjaspinski**

### CODE REVIEW

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]
Add the screenshot also to THEMES.md

**Review by morpheusthewhite — APPROVED** [review]

File: THEMES.md [hunk 1]
```diff
@@ -51,4 +66,6 @@ Here you can find a preview of all the themes. Some of them may have different c
 
 https://user-images.githubusercontent.com/19476925/119483404-84223100-bd87-11eb-8dfa-5af9f7a2e925.mov
 
+## Burnt Sienna
 
+![Burnt Sienna Screenshot](BurntSienna/screenshot.png)
```
Comment (morpheusthewhite) [inline THEMES.md:71]: The name of the theme is a single word.

File: THEMES.md [hunk 2]
```diff
@@ -51,4 +66,6 @@ Here you can find a preview of all the themes. Some of them may have different c
 
 https://user-images.githubusercontent.com/19476925/119483404-84223100-bd87-11eb-8dfa-5af9f7a2e925.mov
 
+## Burnt Sienna
```
Comment (morpheusthewhite) [inline THEMES.md:69]: Theme should be ordered alphabetically

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]
Correct THEMES.md

**Comment by pjaspinski** [comment]
I made both requested changes - theme name is now one word and the screenshot is placed in THEMES.md in alphabetical order.

**Review by morpheusthewhite — APPROVED** [review]

**Pull request merged**